mxd/docs/EDR Transfer Tutorial.md

```diff
@@ -33,6 +33,7 @@ curl --location 'http://localhost/bob/management/edrs' \
 		"policy": {
 			"@type": "odrl:Set",
 			"odrl:permission": {
+			    "odrl:target": "1",
```

Comment:
`target` should not be defined into `permission` according to DSP spec, same for the other ones added in this pr

Reply:
It's needed while initiating negotiation, otherwise negotiation fails with error `Policy in the contract agreement is not equal to the one in the contract offer`.

Reply:
In next release, it's not needed, but  Negotiation `0.6.0` doesn't work without it.
Provider Control Plane logs when `target` field is not part of the permission object.
```
DEBUG 2024-03-04T07:52:50.649246807 DSP: Incoming ContractRequestMessage for class org.eclipse.edc.connector.contract.spi.types.negotiation.ContractNegotiation process
DEBUG 2024-03-04T07:52:51.073577865 [PROVIDER] ContractNegotiation 3bb94506-d68c-4e62-8bf9-700d56400fa9 is now in state REQUESTED.
DEBUG 2024-03-04T07:52:51.605124976 [ProviderContractNegotiationManagerImpl] ContractNegotiation 3bb94506-d68c-4e62-8bf9-700d56400fa9 is now in state AGREEING
DEBUG 2024-03-04T07:52:51.611105437 ContractNegotiation: ID 3bb94506-d68c-4e62-8bf9-700d56400fa9. [Provider] send agreement
DEBUG 2024-03-04T07:52:52.059491099 ContractNegotiation: ID 3bb94506-d68c-4e62-8bf9-700d56400fa9. [Provider] send agreement
SEVERE 2024-03-04T07:52:52.061063278 ContractNegotiation: ID 3bb94506-d68c-4e62-8bf9-700d56400fa9. Fatal error while [Provider] send agreement. Error details: {"@type":"dspace:ContractNegotiationError","dspace:code":"400","dspace:reason":"Contract agreement received. Validation failed: Policy in the contract agreement is not equal to the one in the contract offer","dspace:processId":"42a56a30-aa83-4b90-a5f4-2a8ba2310462","@context":{"@vocab":"https://w3id.org/edc/v0.0.1/ns/","edc":"https://w3id.org/edc/v0.0.1/ns/","tx":"https://w3id.org/tractusx/v0.0.1/ns/","dcat":"http://www.w3.org/ns/dcat#","dct":"https://purl.org/dc/terms/","odrl":"http://www.w3.org/ns/odrl/2/","dspace":"https://w3id.org/dspace/v0.8/"}}
DEBUG 2024-03-04T07:52:52.07563447 [ProviderContractNegotiationManagerImpl] ContractNegotiation 3bb94506-d68c-4e62-8bf9-700d56400fa9 is now in state TERMINATED
```

Reply:
Ok, will exclude it from here and create a new PR.